Config/FilterInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+interface FilterInterface
```

Comment:
i would like to do a phpdoc that explains what a filter is (the definition for one image transformation operation)

Reply:
@dbu could you provide an example ? cause I'm not sure that I understand this comment.

Reply:
```
/**
 * A filter contains the configuration for an image transformation operation.
 *
 * The type of operation is defined by the filter class. The operation parameters are specific to
 * the filter.
 */
```

Reply:
done